README.md

```diff
@@ -224,6 +224,10 @@ If you get the error `Error: NoHypervisorFound` and KVM or mshv is set up then t
 
 For more details on how to verify that KVM is correctly installed and permissions are correct, follow the guide [here](https://help.ubuntu.com/community/KVM/Installation).
 
+### Rust Analyzer
```

Comment:
nit: What do you think about adding this to CONTRIBUTING.md instead?

Reply:
^^^ +1